configuration/src/main/kotlin/com/malinskiy/marathon/config/vendor/apple/ios/XcresultConfiguration.kt

```diff
@@ -3,7 +3,7 @@ package com.malinskiy.marathon.config.vendor.apple.ios
 import com.fasterxml.jackson.annotation.JsonProperty
 
 data class XcresultConfiguration(
-    @JsonProperty("pull") val pull: Boolean = true,
+    @JsonProperty("pullingPolicy") val pullingPolicy: PullingPolicy = PullingPolicy.ALWAYS,
```

Comment:
It might be a good idea to save the old option for at least one release and if pull is enabled default to `ALWAYS`. Later we can remove the `pull` option

Reply:
done.
Also I wanted to leave warning deprecation message, but there is no logger in the configuration module. It might be a good idea in future to take it out from the core module.

Reply:
There is concurrent work around fancy progress bars in the output, that conflicts with the slf4j logger. Maybe as a next step here we can create our own logger interface and depend on that, but i suspect it will break in gradle plugin